### PR TITLE
update website to reflect conf is over and add testimonials

### PR DESCRIPTION
**Pull request by evykassirer**
addresses https://git.uwaterloo.ca/starcon/exec-eyes-only/issues/168

the testimonial tweets take up a fair amount of space on the home page but I'm ok with this because the logistics stuff below that isn't as relevant right now

### CODE REVIEW

File: _pages/cfp.md [hunk 1]
```diff
@@ -7,9 +7,9 @@ permalink: /cfp/
 
 **The call for talk proposals for StarCon 2018 is now closed.**
 
-Decisions will be sent out Tue Oct 31.
+Decisions were sent out Tue Oct 31.
```
Comment (arshiamufti) [inline _pages/cfp.md]: can we just take these lines out?

File: _pages/sample-proj.md [hunk 2]
```diff
@@ -1,56 +0,0 @@
----
-layout: page
```
Comment (arshiamufti) [inline _pages/sample-proj.md]: i'd have preferred this to be taken out in another PR :P but oh well

File: index.md [hunk 3]
```diff
@@ -3,27 +3,40 @@ layout: default
...
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">Today at <a href="https://twitter.com/hashtag/starcon?src=hash&amp;ref_src=twsrc%5Etfw">#starcon</a> I learned so much, all interesting things that I would never have thought to google</p>&mdash; Tristan (@DrTeePot) <a href="https://twitter.com/DrTeePot/status/949754079999270912?ref_src=twsrc%5Etfw">January 6, 2018</a></blockquote>
+
+<br/>
 
-Check out our [invited keynotes and speakers](speakers/#keynotes)! Learn about how you can speak at StarCon [here](/cfp/).
+<script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+{:/}
 
 ## Where and When
```
Comment (arshiamufti) [inline index.md]: move this above the tweets

File: index.md [hunk 4]
```diff
@@ -3,27 +3,40 @@ layout: default
...
 
-StarCon will be held at University of Waterloo on January 6th and 7th, 2018.
+StarCon 2018 was held at University of Waterloo on January 6th and 7th, 2018. Talks were recorded and will be posted once they have been sent to us. Information about a potential StarCon 2019 will be announced soon.
+
+## Who's speaking? Can I apply to speak?
```
Comment (arshiamufti) [inline index.md]: change this to just `Speakers`

File: index.md [hunk 5]
```diff
@@ -3,27 +3,40 @@ layout: default
...
+
+Check out our [2018 invited keynotes and speakers](speakers/#keynotes)! Our call for proposals to speak at StarCon has closed but you can learn more about it [here](/cfp/).
 
 ## Who Can Attend
 
 This conference is open to everyone!
 
 ## Registration
```
Comment (arshiamufti) [inline index.md]: I think we don't need a registration section, just take it out.

**Comment by evykassirer** [comment]
wow thanks for the review from cusec <3

File: _pages/sample-proj.md (hunk 2 above)
Comment (evykassirer) [reply _pages/sample-proj.md]: oh well :p my bad
